region/crateria/central.json

```diff
@@ -775,7 +775,7 @@
           "homeNodes": [ 5 ]
         },
         {
-          "name": "Space Pirate (fighting)",
+          "name": "Green Space Pirate (standing)",
```

Comment:
If those are the two guys at the bottom left during the escape, they're probably the same as in Tourian.
They seem to have 250 health though, not 500. Maybe that's just your data being wrong though.

Reply:
Ffs why did I make them green ones.